### ISSUE TITLE
VideoStreamRequest->createVideo fails to create new video

### ISSUE THREAD

**Issue by hcaz** [ToshY/BunnyNet-PHP#28]
When calling the `createVideo` endpoint as part of the `VideoStreamRequest` it throws an error due to the wrong content-type header being set. According to the [docs](https://docs.bunny.net/reference/video_createvideo) the header should be `'Content-Type' => 'application/*+json'`. This is available within the `Header` enum and just needs changing in the `VideoEndpoint.php` set of constants.

```php
  "content" => array:4 [▼
    "type" => "https://tools.ietf.org/html/rfc7231#section-6.5.13"
    "title" => "Unsupported Media Type"
    "status" => 415
    "traceId" => "00-e85c6c5a7f5df7187fa7438f6282af68-8fdf8a6359a16646-00"
  ]
```

**Comment by ToshY** [comment]
Hey @hcaz 👋 

Good catch, seems that I missed that one. I'll take a look at the PR.

**Comment by ToshY** [comment]
Merged #29
